cg/services/deliver_files/deliver_files_service/deliver_files_service_factory.py

```diff
@@ -104,10 +104,10 @@ def _get_file_fetcher(self, delivery_type: DataDelivery) -> FetchDeliveryFilesSe
 
     @staticmethod
     def _get_sample_file_formatter(
-        workflow: Workflow,
+        app_tag: str,
     ) -> SampleFileFormatter | SampleFileConcatenationFormatter:
         """Get the file formatter service based on the workflow."""
-        if workflow in [Workflow.MICROSALT]:
+        if app_tag in [I, DONT, KNOW, WHICH, ONES, TO, PUT, HERE]:
```

Comment:
all apptags starting with "MW". I would do a list in a constants module and import it here

Reply:
update: all non-archived app tags with "mic" as a prep_category